### PR TITLE
Update eol card text (EXPOSUREAPP-15013)

### PR DESCRIPTION
**Pull request by SamuraiKek**
[Ticket.](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-15013)

Slight change to the new text.

### CODE REVIEW

**Review by OLVI5 — DISMISSED** [review]

File: Corona-Warn-App/src/main/res/values-de/strings.xml
Comment (OLVI5) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:1]: Hello,

please change the abbreviation from BKK to BBK (=Bundesamts für Bevölkerungsschutz und Katastrophenhilfe)

Best Oliver

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -2040,5 +2040,5 @@ Sollten Sie den Test in der App gelöscht haben, können Sie ihn aus dem Papierk
...
+    <string name="home_eol_message">Vielen Dank für die Nutzung der Corona-Warn-App. Während der Pandemie wurden durch die Nutzenden der Corona-Warn-App rund 9 Millionen positive Testergebnisse geteilt. Durch diese Teilung positiver Testergebnisse konnten Personen schnell vor einem erhöhten Infektionsrisiko gewarnt werden, das ihnen sonst unerkannt geblieben wäre. Viele Gewarnte wurden anschließend positiv getestet. Die Corona-Warn-App hat dank Ihrer Mithilfe ihren Zweck der frühzeitigen Warnung erfüllt und die Beendigung von Infektionsketten unterstützt.\n\nAb dem 1. Juni 2023 wird die Corona-Warn-App nicht mehr weiterentwickelt. Sie können weiterhin auf Ihre bereits in der App gespeicherten Zertifikate und das Kontakt-Tagebuch zugreifen. Alle anderen Funktionen stehen nicht mehr zur Verfügung.\n\nWenn Sie wichtige Warnmeldungen für unterschiedliche Gefahrenlagen erhalten wollen, empfehlen wir Ihnen die <a href="https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/NINA-Download/nina-download_node.html">Warn-App NINA des Bundesamts für Bevölkerungsschutz und Katastrophenhilfe (BBK)</a> (external link).</string>
```
Comment (Ein-Tim) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:2043]: Why does this say "external link" in German? It should be "externer Link".

```suggestion
    <string name="home_eol_message">Vielen Dank für die Nutzung der Corona-Warn-App. Während der Pandemie wurden durch die Nutzenden der Corona-Warn-App rund 9 Millionen positive Testergebnisse geteilt. Durch diese Teilung positiver Testergebnisse konnten Personen schnell vor einem erhöhten Infektionsrisiko gewarnt werden, das ihnen sonst unerkannt geblieben wäre. Viele Gewarnte wurden anschließend positiv getestet. Die Corona-Warn-App hat dank Ihrer Mithilfe ihren Zweck der frühzeitigen Warnung erfüllt und die Beendigung von Infektionsketten unterstützt.\n\nAb dem 1. Juni 2023 wird die Corona-Warn-App nicht mehr weiterentwickelt. Sie können weiterhin auf Ihre bereits in der App gespeicherten Zertifikate und das Kontakt-Tagebuch zugreifen. Alle anderen Funktionen stehen nicht mehr zur Verfügung.\n\nWenn Sie wichtige Warnmeldungen für unterschiedliche Gefahrenlagen erhalten wollen, empfehlen wir Ihnen die <a href="https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/NINA-Download/nina-download_node.html">Warn-App NINA des Bundesamts für Bevölkerungsschutz und Katastrophenhilfe (BBK)</a> (externer Link).</string>
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Please fix the language

File: Corona-Warn-App/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/values-de/strings.xml]: @Ein-Tim, thanks for spotting it. iOS has been informed as well.

Comment (Ein-Tim) [reply Corona-Warn-App/src/main/res/values-de/strings.xml]: Thank you!

**Pull request merged**